apps/workers/sql-load/main.go

```diff
@@ -16,7 +17,9 @@ type options struct {
 	Postgres bootstrap.DBOptions `toml:"postgres"`
 	MySQL    bootstrap.DBOptions `toml:"mysql"`
 
-	sqlDB    *sql.DB
+	sqlDB  *sql.DB
+	sqlxDB *sqlx.DB // used for running direct exec command
```

Comment:
Do we need to have two separate connections to the db?

Reply:
the sqlx just uses the existing sql connection. I'm not sure if that creates a separate connection pool, but I didn't want to break the batch loader.